mesh/mesh_test.go

```diff
@@ -66,6 +66,14 @@ func createTestMesh(t *testing.T) *testMesh {
 	exec := NewExecutor(db, atxsdata, tm.mockVM, tm.mockState, lg)
 	msh, err := NewMesh(db, atxsdata, tm.mockClock, tm.mockTortoise, exec, tm.mockState, lg)
 	require.NoError(t, err)
+	ctx, cancel := context.WithCancel(context.Background())
+	done := make(chan struct{})
+	go func() {
+		msh.Start(ctx)
+		close(done)
+	}()
+	t.Cleanup(func() { <-done })
+	t.Cleanup(cancel)
```

Comment:
```suggestion
	ctx, cancel := context.WithCancel(context.Background())
	defer cancel()
	var eg errgroup.Group
	eg.Go(func() error {
		mesh.Start(ctx)
		return nil
	})
	t.Cleanup(func() { assert.NoError(t, eg.Wait()) } )
```

Reply:
the error group will never return an error so why is the assert needed?

Reply:
Yeah, it's not. Just a force of habit from other tests 🙂